unit_tests/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ set(
   with_durability
   json_apis_from_client
   zone_report
+  test_host_list_context_string
```

Comment:
For consistency with other unit tests, this should be `host_list_context_string`, and `irods_test_host_list_context_string.cmake` should be named `irods_host_list_context_string.cmake`. Our current naming scheme only puts `test_` in the `.cpp` filename.

Reply:
done

Reply:
done